astropy/units/imperial.py

```diff
@@ -78,6 +76,15 @@
          doc="nautical unit of speed: 1 nmi per hour")
 
 
+###########################################################################
+# FORCE
+
+def_unit('lbf', 32.174049 * lb * ft * si.s**-2, namespace=_ns,
+         doc="Pound: force")
+def_unit(['kip', 'kilopound'], 1000 * lbf, namespace=_ns,
+         doc="Kilopound: force")
```

Comment:
Many of the SI units use the `prefixes` argument to implement this.  You can give it a specific list of prefixes (see the ``units/core.py`` file, look for the ``si_prefixes`` variable for detailed format) if you want.  The downside here is that it then has to still end in "lbf", but maybe that's less confusing? (i.e., "klbf" or "kilbf" instead of "kip".)

Or is "kip" a standard abbreviation in some area?  I've never encountered it, but admittedly I rarely encounter imperial units at all in science work...